Pod/Classes/Visualizer.swift

```diff
@@ -1,204 +1,54 @@
 //
 //  TouchVisualizer.swift
-//  Pods
+//  TouchVisualizer
 //
 //  Created by MORITA NAOKI on 2015/01/24.
 //  Copyright (c) 2015年 molabo. All rights reserved.
 //
 
-import UIKit
+import Foundation
 
 final public class Visualizer {
     
-    private var config: Configuration!
-    private var touchViews = [TouchView]()
-    private var enabled:Bool = false
-    
-    private var previousLog = ""
-    
-    static let sharedInstance = Visualizer()
+    // MARK: - Public Variables
+    static public let sharedInstance = Visualizer()
+    public var enabled = false
+    public var config: Configuration!
+    public var touchViews = [TouchView]()
+    public var previousLog = ""
     
+    // MARK: - Object life cycle
     private init() {
-        NSNotificationCenter.defaultCenter().addObserver(self, selector: "orientationDidChangeNotification:", name: UIDeviceOrientationDidChangeNotification, object: nil)
-        NSNotificationCenter.defaultCenter().addObserver(self, selector: "applicationDidBecomeActiveNotification:", name: UIApplicationDidBecomeActiveNotification, object: nil)
-        UIDevice.currentDevice().beginGeneratingDeviceOrientationNotifications()
+        NSNotificationCenter
+            .defaultCenter()
+            .addObserver(self, selector: "orientationDidChangeNotification:", name: UIDeviceOrientationDidChangeNotification, object: nil)
+        
+        NSNotificationCenter
+            .defaultCenter()
+            .addObserver(self, selector: "applicationDidBecomeActiveNotification:", name: UIApplicationDidBecomeActiveNotification, object: nil)
+        
+        UIDevice
+            .currentDevice()
+            .beginGeneratingDeviceOrientationNotifications()
         
         warnIfSimulator()
     }
     
     deinit {
-        NSNotificationCenter.defaultCenter().removeObserver(self)
+        NSNotificationCenter
+            .defaultCenter()
+            .removeObserver(self)
     }
     
-    @objc func applicationDidBecomeActiveNotification(notification: NSNotification) {
+    // MARK: - Helper Functions
+    @objc public func applicationDidBecomeActiveNotification(notification: NSNotification) {
         UIApplication.sharedApplication().keyWindow?.swizzle()
     }
     
-    @objc func orientationDidChangeNotification(notification: NSNotification) {
+    @objc public func orientationDidChangeNotification(notification: NSNotification) {
         let instance = Visualizer.sharedInstance
         for touch in instance.touchViews {
             touch.removeFromSuperview()
         }
     }
 }
-
```

Comment:
"Cleanliness is Next to Happiness"

I personally prefer them in separate files. (I do this with all of my extensions)